### PR TITLE
DCS-1392 Adding link to allow overriding incoming arrival

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: integration_tests/integration/bookedtoday/arrivals/isSingleMatch.spec.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ import expectedArrivals from '../../../mockApis/responses/expectedArrivals'
 import SexPage from '../../../pages/bookedtoday/arrivals/confirmArrival/sexPage'
```
Comment (andrewrlee) [inline integration_tests/integration/bookedtoday/arrivals/isSingleMatch.spec.ts:8]: Rename to be scenario specific

File: integration_tests/integration/bookedtoday/arrivals/isSingleMatch.spec.ts [hunk 2]
```diff
@@ -82,21 +83,10 @@ context('Check Answers', () => {
...
-      })
-    confirmAddedToRollPage
-      .backToDigitalPrisonServices()
-      .should('contain', 'Back to Digital Prison Services')
-      .should('have.attr', 'href')
-      .then(href => {
-        expect(href).to.equal('https://digital-dev.prison.service.justice.gov.uk')
-      })
+    confirmAddedToRollPage.viewEstablishmentRoll().exists()
+    confirmAddedToRollPage.backToDigitalPrisonServices().exists()
```
Comment (andrewrlee) [inline integration_tests/integration/bookedtoday/arrivals/isSingleMatch.spec.ts:87]: moved these into the page using new util function

File: integration_tests/integration/bookedtoday/arrivals/isSingleMatch.spec.ts [hunk 3]
```diff
@@ -122,23 +112,9 @@ context('Check Answers', () => {
       expectedArrival.id,
       SingleMatchingRecordFoundPage
     )
-    singleMatchingRecordFoundPage.continue().click()
```
Comment (andrewrlee) [inline integration_tests/integration/bookedtoday/arrivals/isSingleMatch.spec.ts:125]: this test wasn't doing much so got rid

File: server/views/pages/bookedtoday/arrivals/singleMatchingRecordFound.njk [hunk 4]
```diff
@@ -3,48 +3,47 @@
 
 {% block content %}
     <div class="govuk-width-container">
-        <main class="govuk-main-wrapper">
```
Comment (andrewrlee) [inline server/views/pages/bookedtoday/arrivals/singleMatchingRecordFound.njk:6]: got rid of these wrappers as they already exists in layout

File: integration_tests/integration/bookedtoday/arrivals/hasNoMatchAndPncOnly.spec.ts [hunk 5]
```diff
@@ -105,4 +108,29 @@ context('No match found', () => {
       value().contains('20 September 1982')
     }
   })
+
+  it('Can process arrival', () => {
```
Comment (andrewrlee) [inline integration_tests/integration/bookedtoday/arrivals/hasNoMatchAndPncOnly.spec.ts:112]: Moved this test over as this now falls under this scenario - with its previous owner now falling under the isSingleMatch

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**